client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1768,7 +1768,7 @@
                     if (angular.isDefined(item)) {
                         item._datelinedate = '';
 
-                        if (angular.isDefined(item.dateline.located) && !_.isNull(item.dateline.located)) {
+                        if (item.dateline && item.dateline.located != null) {
```

Comment:
I think the idea here is to compare the value both to `undefined` and `null` at the same time (i.e. in a single comparison), although relying on Javascript's type coercion can be tricky and should probably be avoided by default (not sure if our linter is configured to issue a warning on that, though).

```js
>>> x = undefined
>>> x == null
true
>>> x = null
>>> x == null
true
```
Would it perhaps also be fine to simply check if the value is truthy?

```js
if (... && item.dateline.located) { ...
```